# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/base/nsRefreshDriver.h
# Commit: 20a5c1f3b35e
# Full Hash: 20a5c1f3b35e80849e0b98ffbe3f9488d703717b
# Author: Frederic Wang <fred.wang@free.fr>
# Date: 2023-11-22 21:50:37
# Regressor Bug: 1807253
# File Overlap Count: 1
# Description:
#   Bug 1807253 - unreliable timing for content-visibility:auto, r=emilio
#   
#   Rewrite implementation of content-visibility: auto as defined in
#   https://github.com/w3c/csswg-drafts/issues/8542
#   
# ==============================================================================

diff -r b90083f581e1 -r 20a5c1f3b35e layout/base/nsRefreshDriver.h
--- a/layout/base/nsRefreshDriver.h	Mon Nov 20 11:33:16 2023 +0000
+++ b/layout/base/nsRefreshDriver.h	Wed Nov 22 13:00:36 2023 +0000
@@ -499,7 +499,8 @@
   void RunFrameRequestCallbacks(mozilla::TimeStamp aNowTime);
   void UpdateIntersectionObservations(mozilla::TimeStamp aNowTime);
   void UpdateRelevancyOfContentVisibilityAutoFrames();
-  MOZ_CAN_RUN_SCRIPT void NotifyResizeObservers();
+  MOZ_CAN_RUN_SCRIPT void
+  DetermineProximityToViewportAndNotifyResizeObservers();
   void MaybeIncreaseMeasuredTicksSinceLoading();
   void EvaluateMediaQueriesAndReportChanges();
 